### DISCUSSION TITLE
Broader question, about SSR, Cache and prerender - Partial cache/prerender and Server Side Hybridization

### DISCUSSION THREAD

**Discussion by lukaszpolowczyk** [sveltejs/kit#5288] [Q&A]
A situation like this:
1. a page with entries, updated once a day. Page with posts (along with template) rendered once per day, and hosted on a cheap static page host.
2. data such as post likes and comments - dynamically fetched and rendered on the user side.

But what about the situation when:
1. as above
2. data such as post likes and comments - dynamically fetched and rendered on the server side?
That is: Static page + dynamically added small data.

It would be sort of Partial cache/prerender and (almost like) **Server Side Hybridization**.

Is such a thing within SvelteKit doable?
What I mean by this is that you don't need to create your own mechanism to combine a static daily page with dynamic data.

**Comment by manvankaliber** [comment]
You can [mark a page prerenderable](https://kit.svelte.dev/docs/page-options#prerender) and use `<iframe>` inside that page to load in dynamic data from a non static route. I've done this in one of my projects. It works very well.

**Reply by lukaszpolowczyk** [reply]
But in what way does the iframe?
Does it use SvelteKit in any way?

What I was thinking about was that this static page, is a page prepared by SvelteKit, ready for SvelteKit to add an additional portion of data to it.

**Reply by manvankaliber** [reply]
> But in what way does the iframe? Does it use SvelteKit in any way?
> 
> What I was thinking about was that this static page, is a page prepared by SvelteKit, ready for SvelteKit to add an additional portion of data to it.

If you put the iframe on a prerendered page and point the iframe to a dynamic SSR page in svelte-kit.

But to just answer your question: svelte-kit doesn't have an option to prerender a page partially.
